backend/api/app/application/services/user_service.py

```diff
@@ -76,3 +107,152 @@ def verifySSID(self, ssid: str, ip_address: str):
             return True
         else:
             return False
+        
+    def logout_user(self, ssid: str, ip_address: str):
+            try:
+                if not ssid:
+                    return {"message": "SSID is required", "status": 400}
+
+                try:
+                    ses_id = uuid.UUID(ssid)
+                except ValueError:
+                    return {"message": "Invalid SSID format", "status": 400}
+
+                session = SessionRepository.get_active_by_id(ses_id, ip_address)
+                if not session:
+                    return {"message": "Session not found", "status": 404}
+
+                session.logged_out = True
+                SessionRepository.update(session)
+
+                return {"message": "User logged out successfully", "status": 200}
+
+            except ValueError as ve:
+                return {"message": str(ve), "status": 400}
```

Comment:
Sending user unhandled ValueError or Exception is possible security hazard because of the stacktrace that user will probably get. We should not ever send back generic error messages if we are not sure that they're thrown by us.